test/query.test.js

```diff
@@ -848,6 +848,19 @@ describe('query', () => {
     })
   })
 
+  test.only('create_access_provider', async () => {
+    // TODO: Define or remove this test
+    const accessProvider = await client.query(
```

Comment:
@nathaniel-may @marrony Any advice re: how I can go about testing this? I'm assuming I need to authenticate or point this to a different URL cause right now I get a ` PermissionDenied: permission denied` error.

Reply:
you need to use a admin key to create access providers

Reply:
@marrony Thanks. I figured it was something like that. Would something like this work?

```javascript
    const accessProvider = await client.query(
      query.CreateAccessProvider({
        name: 'my-provider',
        issuer: 'ryan',
        jwks_url: 'https://db.fauna.com',
      }), { secret: ADMIN_KEY}
    )
```